auth/src/main/resources/application.yml

```
@@ -30,9 +30,37 @@ eureka:
+    enable:
+      jvm: true
+  zipkin:
+    tracing:
+      endpoint: "http://${ZIPKIN_HOST:localhost}:${ZIPKIN_PORT:9411}/api/v2/spans"
+  tracing:
+    sampling:
+      probability: ${ZIPKIN_SAMPLING_PROBABILITY:1.0}
+  prometheus:
+    metrics:
+      export:
+        enabled: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)